src/transforms/select.jl

```diff
@@ -2,6 +2,38 @@
 # Licensed under the MIT License. See LICENSE in the project root.
 # ------------------------------------------------------------------
 
+struct TableSelection{T} 
+  table::T
+  cols::Vector{Symbol}
+end
+
+Tables.istable(::Type{<:TableSelection}) = true
+Tables.columnaccess(::Type{<:TableSelection}) = true
+Tables.columns(ts::TableSelection) = ts
+Tables.columnnames(ts::TableSelection) = ts.cols
+Tables.getcolumn(ts::TableSelection, col::Int) =
+  Tables.getcolumn(ts.table, ts.cols[col])
+Tables.getcolumn(ts::TableSelection, col::Symbol) = 
+  Tables.getcolumn(ts.table, col)
```

Comment:
Let's call the argument name `t`, the name `ts` can be associated with a collection of objects like we do with `xs` and `ys` for example.

Reply:
Done!